### PR TITLE
Add error message response to rpc-server

### LINKED ISSUE #248: Improve rpc server error codes and messages
When running rpc server, if the request the user send fails then RPC server returns error code 400 and no message when any error is thrown.
## Expected Behavior

There are some cases when rpc server should return an error message and a more relevant code such as:
- sdk is deadlocked
- request is too big
## Current Behavior

Thrown errors always respond 400 status code and no message

### PR DESCRIPTION
**Pull request by ItsLuized**
This pull request adds error message response to `rpc-server`

To add error message response we must add the error message from `SDK` to the `rpc-server` response. This error messages will help debugging `rpc-server` when requests fail.

Closes #248

### CODE REVIEW

**Comment by diegoalzate** [comment]
we did not update status codes, only messages

File: packages/sdk/src/index.ts [hunk 1]
```diff
@@ -69,6 +68,8 @@ export default class SDK {
   public deadlockTimestamp: number | undefined;
   // toggle to not select entry nodes while another one is being selected
   private selectingEntryNode: boolean | undefined;
+  // max number of segments sdk can send to entry node
+  public maximumSegmentsPerRequest: number = 100;
```
Comment (nionis) [inline packages/sdk/src/index.ts:72]: this change seems out of scope

**Review by nionis — APPROVED** [review]
